### DISCUSSION TITLE
CatServer兼容性报告中的等待MiraiMC1.6应该改成使用MiraiMC1.6+了

### DISCUSSION THREAD

**Discussion by Muska-Ami** [DreamVoid/MiraiMC#204] [建议 / Ideas]
如题(
https://wiki.miraimc.dreamvoid.ml/troubleshoot/compatibility-report/catserver.html

**Comment by Muska-Ami** [comment]
后补：Cauldron端的也要改了

**Comment by DreamVoid** [comment]
https://github.com/MiraiMC/MiraiMC-Wiki/ 那就改吧

**Reply by Muska-Ami** [reply]
已经开PR了

**Comment by Muska-Ami** [comment]
居然没发现还有个文档库
